src/current/v22.2/backup-architecture.md

```diff
@@ -91,6 +91,30 @@ The backup metadata files describe everything a backup contains. That is, all th
 
 With the full backup complete, the specified storage location will contain the backup data and its metadata ready for a potential [restore](restore.html). After subsequent backups of the `movr` database to this storage location, CockroachDB will create a _backup collection_. See [Backup collections](take-full-and-incremental-backups.html#backup-collections) for information on how CockroachDB structures a collection of multiple backups.
 
+## Backup jobs with locality
```

Comment:
v22.2 did not have any technical detail on locality-aware backups, this is taken from v23.1 / v23.2

Reply:
Do we need to update 22.2? It's EOL, right?

Reply:
Not quite... in June it's EOL. I was really playing it safe here updating v22.2 as well.

Reply:
ah, it's EOL for me but not for thee. :) Thank you!